### ISSUE TITLE
Footer links becomes column too soon

### ISSUE THREAD

**Issue by hustlerZzZ** [facebook/stylex#165]
Footer container links that contain develop, explore, participate and other becomes column too soon it should be totally response first it becomes 2 and then 1 which is good according to all screen sizes.

We can easily fix this by adding more media queries and setting properties according to this.

I can easily fix this. Kindly assign this to me.

![image](https://github.com/facebook/stylex/assets/102817420/0b3219f0-40f7-4a41-949c-c25a557e1532)

**Comment by nmn** [comment]
I don't think this needs to be done. I just tested and it wraps when I would expect it to.
